modules/mixcr/1.0/mixcr.smk

```diff
@@ -70,6 +70,8 @@ rule _mixcr_run:
     input:
         fastq_1 = rules._mixcr_input_fastq.output.fastq_1,
         fastq_2 = rules._mixcr_input_fastq.output.fastq_2,
+        fastq_1_real = CFG["inputs"]["sample_fastq_1"], # Prevent premature deletion of fastqs marked as temp
+        fastq_2_real = CFG["inputs"]["sample_fastq_2"],
```

Comment:
sorry, I didn't get how this prevents premature deletion of fastq files. Is it because they are referenced in config? They don't seem to be referenced in the rules further down.

Reply:
Snakemake deletes a temp file after all the rules that use that file have been completed. If the real fastq file is only referenced during the rule `_mixcr_input_fastq`, the would be deleted as soon as the symlink is created. To get around that I specify them as inputs to the actual `_mixcr_run` rule, even though they aren't used in the command. That way they won't get deleted until after the mixcr run is finished.